presto-main/src/test/java/com/facebook/presto/operator/aggregation/differentialentropy/TestUtils.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.operator.aggregation.differentialentropy;
+
+import org.testng.annotations.Test;
+
+import java.util.Random;
+
+import static com.facebook.presto.operator.aggregation.differentialentropy.EntropyUtils.calculateEntropyFromSamples;
+import static org.testng.Assert.assertEquals;
+
+public class TestUtils
```

Comment:
This name suggests that the class contains utility methods useful for testing. However, this is not the case. I suggest to rename this class to rename EntropyUtils to EntropyCalculator and rename this class to TestEntropyCalculator. I also suggest to rename calculateEntropyFromSamples to calculate.

```
public class TestEntropyCalculator
{
    @Test
    public void testUniformDistribution()
    {
        Random random = new Random(13);
        double[] samples = new double[10_000_000];
        for (int i = 0; i < samples.length; i++) {
            samples[i] = random.nextDouble();
        }
        assertEquals(calculate(samples), 0, 0.02);
    }

    @Test
    public void testNormalDistribution()
    {
        Random random = new Random(13);
        double[] samples = new double[10_000_000];
        double sigma = 0.5;
        for (int i = 0; i < samples.length; i++) {
            samples[i] = 5 + sigma * random.nextGaussian();
        }
        assertEquals(calculate(samples), 0.5 * Math.log(2 * Math.PI * Math.E * sigma * sigma) / Math.log(2), 0.02);
    }
}
```